src/resources/Abathur.ts

```diff
@@ -149,33 +149,6 @@ export class Abathur {
 		return this._globalAssets;
 	}
 
-	private canReceiveBasicMineralsForReaction(mineralQuantities: { [resourceType: string]: number },
```

Comment:
why remove these two methods?

Reply:
Both of these functions assume that something is going to either send or buy minerals for this room. I don't see how that happens and there's no docs that say where it happens and I observed it not happening, so I assumed they simply didn't work and removed them. If you can point out where the code is that does these things, I'll add comments to point it out and investigate why these returned true while the corresponding systems didn't actually work.